scripts/competitiond.py

```diff
@@ -743,14 +748,9 @@ def run_once(self):
 
         if not self.leaderboard_only:
             for submit_bundle in submissions:
-                if 'user_name' in submit_bundle['owner']:
-                    logger.info(
-                        "Mimicking submission for " "{owner[user_name]}".format(**submit_bundle)
-                    )
-                else:
-                    logger.info(
-                        "Mimicking submission for " "{metadata[description]} (no username)".format(**submit_bundle)
-                    )
+                logger.info(
```

Comment:
@robinjia I removed the `user_name` check because there seem to be many other places where we just assume that "user_name" exists, and I can't think of a way to replicate this error. Let me know if you have an idea about how to replicate this, and if so, we can add it back in.